modules/_labs/pagination/react/lib/Pagination/common/utils/space.ts

```diff
@@ -0,0 +1,109 @@
+import {spacing} from '@workday/canvas-kit-react-core';
+
+export type Spacing = keyof typeof spacing;
+
+export interface SpaceProps {
```

Comment:
When we move this to common, we should make sure to deprecate the one in labs-core. We may want to do that as part of v5 TBH

Reply:
Yeah, all of the layout components will be v5 so we can cleanly make that switch.